README.md

```diff
@@ -136,15 +137,17 @@ To avoid specifying `tasmota_commands` for each host using host_vars you can use
 
 Sets the `TelePeriod` to 10 seconds to all devices specified in the `tasmota` host group.
 
-	- hosts: tasmota_devices  
+	- name: Setup tasmota devices with role
+      hosts: tasmota_devices  
 	  # disable fact gathering since this is currently not possible on tasmota devices  
-      gather_facts: no
+      ansible.builtin.gather_facts: false
       vars:
         tasmota_commands:
             - command: TelePeriod
               value: 10
 	  roles:
 	    - tobias_richter.tasmota
+          delegates_to: localhost
```

Comment:
No that is not correct. The action plugin will call the API of the inventory host. The existing documentation is correct.

Reply:
Correct would also be `delegate_to`, see also the ansible documentation: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_delegation.html#delegating-tasks

Reply:
If you would have tested your documentation changes you would have seen that it is not working with delegation to localhost:
```
PLAY [tasmota] ******************************************************************************************************

TASK [tobias_richter.tasmota : Configure tasmota.] ******************************************************************
Thursday 07 December 2023  22:00:53 +0100 (0:00:00.158)       0:00:00.158 *****
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /cm?user=&password=&cmnd=NtpServer1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3203cb8f40>: Failed to establish a new connection: [Errno 111] Connection refused'))
fatal: [socket055]: FAILED! => {"msg": "Unexpected failure during module execution.", "stdout": ""}

PLAY RECAP **********************************************************************************************************
socket055     : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

Thursday 07 December 2023  22:00:53 +0100 (0:00:00.041)       0:00:00.200 *****
===============================================================================
tobias_richter.tasmota : Configure tasmota. ------------------------------------------------------------------ 0.04s
```

Reply:
Happy new year :D

I tested my configuration changes, but i set `ansible_host` in the inventory and it worked.
Nevermind, you are  correct, `delegate_to` is not needed.
I deleted the documentation changes.